packages/core/contracts-ovm/insured-bridge/implementation/BridgeDepositBox.sol

```diff
@@ -31,12 +31,12 @@ interface TokenLike {
  * @notice Accepts deposits on Optimism L2 to relay to Ethereum L1 as part of the UMA insured bridge system.
  */
 
-contract BridgeDepositBox is Legacy_Testable, Legacy_Lockable {
+abstract contract BridgeDepositBox is Legacy_Testable, Legacy_Lockable {
```

Comment:
Does this _have_ to be `abstract` or are you just changing it to emphasize that it shouldn't be used in production?

Reply:
the latter. you should not be deploying the `BridgeDepositBox` on its own.